src/probnum/linalg/solvers/_probabilistic_linear_solver.py

```diff
@@ -231,7 +236,8 @@ class BayesCG(ProbabilisticLinearSolver):
 
     def __init__(
         self,
-        stopping_criterion: stopping_criteria.LinearSolverStoppingCriterion = stopping_criteria.MaxIterationsStoppingCriterion()
+        stopping_criterion: stopping_criteria.LinearSolverStoppingCriterion =
+stopping_criteria.MaxIterationsStoppingCriterion()
```

Comment:
This does not seem compatible with the style determined by our code formatter `black`. The `InvocationError` means there is a piece of code which is not correctly formatted. I suggest you run `tox -re format`. This should reformat this line correctly and satisfy `black`.

Reply:
Done! Reformatted code using black and made a new commit for the changes.

Reply:
Now that the formatting problem is fixed, but tests are raising errors :(
It looks like the problems are all with installling dependencies, which may be caused by installing the dependencies with `pip` rather than `pip3` as in https://stackoverflow.com/questions/57235210/i-have-problem-while-installing-requirement-txt. 
But installing dependencies with `pip` seems to be working fine in previous commits, so I am not sure whether this is the problem.

Reply:
> Also, if we reformat the code as black suggested, i.e. put stopping_criterion: stopping_criteria.LinearSolverStoppingCriterion = stopping_criteria.MaxIterationsStoppingCriterion() in a single line, pylint would be raising line-too-long error for not breaking the long line. It looks like black formatting and pylint formatting are having a conflict here.

Here we'll just have to make a compromise and put in a pylint ignore. Our class names for the linear solvers are just too long to satisfy both black and pylint. So an ignore at that point is totally fine.

Thanks for taking care of this so diligently!

Reply:
Thank you for replying! I think good formatting in any project plays an important role in enhancing readability and ease of understanding, and it is important for a contributor to maintain it, so I am a bit perfectionist on this issue.

> Here we'll just have to make a compromise and put in a pylint ignore.

That's what I think as well, for class names to be meaningful they must be long. So we have to make a compromise and put a pylint ignore at the start of that file.